views/layout.html

```diff
@@ -64,4 +65,18 @@
     <br-app></br-app>
     {% endblock %}
   </body>
+  {% else %}
+  <body>
+    {% block unsupportedUserAgent %}
+    <div class="container">
+      <div class="alert alert-danger">
```

Comment:
Remove bootstrap classes; doesn't have to look pretty and bootstrap isn't required at this level.